src/System.Security.Cryptography.Xml/tests/KeyInfoX509DataTest.cs

```diff
@@ -147,14 +147,15 @@ public void AddCertificate_Null()
         public void AddIssuerSerial_Null_Serial()
         {
             KeyInfoX509Data data = new KeyInfoX509Data();
-            Assert.Throws<FormatException>(() => data.AddIssuerSerial(null, "serial"));
+            Assert.Throws<ArgumentException>(() => data.AddIssuerSerial(null, "serial"));
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "https://github.com/dotnet/corefx/issues/18690")]
         public void AddIssuerSerial_Issuer_Null()
         {
             KeyInfoX509Data data = new KeyInfoX509Data();
-            Assert.Throws<ArgumentNullException>(() => data.AddIssuerSerial("issuer", null));
+            Assert.Throws<ArgumentException>(() => data.AddIssuerSerial("issuer", null));
```

Comment:
@bartonjs - I changed this because logic for AddIssuerSerial got unecessarily too complicated with all ArgumentNullException vs ArgumentException